tracker/settings/base.py

```diff
@@ -63,6 +63,8 @@
     'wagtail.images',
     'wagtail.search',
     'wagtail.admin',
+    # See https://docs.wagtail.io/en/stable/reference/contrib/legacy_richtext.html#legacy-richtext
+    'wagtail.contrib.legacy.richtext',
```

Comment:
👍 

(It would be cool to get off of this someday! Relying on `.rich-text` for styling was always a bit hacky...)